src/services/legal-services.ts

```diff
@@ -436,10 +455,11 @@ export default class LegalServices {
    */
   static async sendCredentialOffer (
     businessId: string,
-    credentialType: DigitalCredentialTypes)
+    credentialType: DigitalCredentialTypes,
+    preconditionsResolved?: { selfAttestedRoles: string[] })
     : Promise<AxiosResponse> {
     const url = `businesses/${businessId}/digitalCredentials/${credentialType}`
-    return axios.post(url)
+    return axios.post(url, { preconditionsResolved })
```

Comment:
Looks like this is a dependency on the Legal API. Just need to keep note of this for deployment purposes.

Reply:
`attestCredentialConnection` above also requires the latest API changes to function.

Reply:
As a result of these dependencies, I recommend that we don't merge this PR yet, as it would automatically makes its way to Dev, Test and Prod, which I think would break the current feature.

Thoughts?

Reply:
The LEAR PRs have now been merged. I'll resolve tis conversation but we need to be aware to deploy this (Filings UI) at the same time as LEAR.